### DISCUSSION TITLE
23.12.11 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#85] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
  - 오늘은 개발을 못 했습니다..ㅜㅜ
- 내일은 어떤 일을 하나요?
  - PR 올리기
  - 성능 테스트에 관한 논의..?

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 매핑 테이블 객체 수정
- 내일은 어떤 일을 하나요?
  - 테스트 작성
